docs/companion.md

```diff
@@ -563,6 +563,17 @@ app.use(
 When signing on the client, this function will only be called for multipart
 uploads.
 
+##### `s3.acl`
```

Comment:
The companion docs put the env var and JS option in the same header. This option already exists: https://uppy.io/docs/companion/#companion_aws_acl so you should add the JS var in front of it (see other options).

Or maybe the env var should be moved here?